packages/output/src/common/output-channel.ts

```diff
@@ -20,29 +20,29 @@ import URI from '@theia/core/lib/common/uri';
 import { Deferred } from '@theia/core/lib/common/promise-util';
 import { Resource, ResourceResolver } from '@theia/core/lib/common/resource';
 import { CommandRegistry, CommandContribution } from '@theia/core/lib/common/command';
-import { OpenerService, open, QuickPickService } from '@theia/core/lib/browser';
+import { QuickPickService } from '@theia/core/lib/browser';
 import { Emitter, Event, Disposable, DisposableCollection } from '@theia/core';
 import { QuickPickItem } from '@theia/core/lib/common/quick-pick-service';
 import { MonacoEditorModel } from '@theia/monaco/lib/browser/monaco-editor-model';
 import { MonacoTextModelService, IReference } from '@theia/monaco/lib/browser/monaco-text-model-service';
 import { OutputUri } from './output-uri';
-import { OutputCommands } from '../browser/output-contribution';
+import { OutputCommands, OutputContribution } from '../browser/output-contribution';
 import { OutputResource } from '../browser/output-resource';
 import { OutputPreferences } from './output-preferences';
 import { OutputConfigSchema } from './output-preferences';
 
 @injectable()
 export class OutputChannelManager implements CommandContribution, Disposable, ResourceResolver {
 
-    @inject(OpenerService)
-    protected readonly openerService: OpenerService;
-
     @inject(QuickPickService)
     protected readonly quickPickService: QuickPickService;
 
     @inject(MonacoTextModelService)
     protected readonly textModelService: MonacoTextModelService;
 
+    @inject(OutputContribution)
+    protected readonly outputContribution: OutputContribution;
```

Comment:
I don't think that channel should know anything about contribution managing it

Reply:
Could you give some pointers besides saying you do not like this approach? Thanks!

Reply:
sorry, I commented here: https://github.com/eclipse-theia/theia/pull/8243#discussion_r461374543